modules/storage-api/src/main/java/org/apache/ignite/internal/storage/index/StorageSortedIndexDescriptor.java

```diff
@@ -36,9 +36,7 @@
  * @see SortedIndexStorage
  */
 public class StorageSortedIndexDescriptor implements StorageIndexDescriptor {
-    /**
-     * Descriptor of a Sorted Index column (column name and column sort order).
-     */
+    /** Descriptor of a Sorted Index column (column name and column sort order). */
```

Comment:
Please revert all these useless javadoc updates

Reply:
I didn't see anywhere that this was prohibited.

Reply:
Fix it